src/poliastro/plotting.py

```diff
@@ -57,29 +55,36 @@ def __init__(self, ax=None, num_points=100):
         self.num_points = num_points
         self._frame = None
         self._attractor_radius = None
+        self.orbits = list(tuple())  # type: List[Tuple[poliastro.twobody.orbit.Orbit], str]
```

Comment:
Thanks for adding the type string!

Reply:
On the other hand: shouldn't this be `self._orbits`, with optionally a `@property` to give read-only access?

Reply:
You are righ! Changing.